### PR TITLE
Fix icons that should not be flipped

### PR DESCRIPTION
**Pull request by huayunh**
#### Changes proposed in this Pull Request:
- Flip only icons with directionality

### CODE REVIEW

**Comment by jeffvg** [comment]
I'm running both. looks like you might have a permissions thing going on? "operation not permitted"

**Comment by huayunh** [comment]
Looks like another catalina issue

**Comment by huayunh** [comment]
Alright after some system preference update everything seems fine

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## vX.X.X (to be published)
+
+_*Breaking Changes:*_
```
Comment (joebochill) [inline CHANGELOG.md:5]: I would not consider this a breaking change...aside from the demos, we're only (effectively) unflipping the search icon in the Header bar.

**Review by joebochill — APPROVED** [review]

**Pull request merged**